nautobot/extras/models/jobs.py

```diff
@@ -825,6 +860,8 @@ def to_cron(self):
             return schedules.crontab(minute=t.minute, hour=t.hour)
         elif self.interval == JobExecutionType.TYPE_WEEKLY:
             return schedules.crontab(minute=t.minute, hour=t.hour, day_of_week=t.weekday())
+        elif self.interval == JobExecutionType.TYPE_CUSTOM:
+            return self.get_crontab()
```

Comment:
Just to be pedantic, in the current implementation if `crontab` is malformed, this would return the value of an exception, and not a properly-parsed crontab object, which I do not think is what we would actually want.

Reply:
yeah, changing the get_crontab() method to only return a crontab() would fix this